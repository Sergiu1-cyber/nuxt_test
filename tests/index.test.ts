import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import Home from "../pages/index.vue"

describe('componentul e vizibil', () => {
  test('componentul e pregatit', () => {
    const wrapper = mount(Home)
    expect(wrapper.text()).toMatch(/home page/i)
    expect(wrapper.html()).toMatchSnapshot()
  })
})

