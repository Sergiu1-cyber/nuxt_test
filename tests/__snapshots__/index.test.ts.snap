// Vitest Snapshot v1

exports[`componentul e vizibil > componentul e pregatit 1`] = `
"<div>
  <h1 class=\\"text-red-500\\">Home Page</h1>
</div>"
`;
